personal_assistant/pa_note/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class Note(models.Model):
+    name = models.CharField(max_length=50, null=False)
+    description = models.CharField(max_length=150, null=False)
+    done = models.BooleanField(default=False)
+    created = models.DateTimeField(auto_now_add=True)
+
+
+    def __str__(self):
+        return f"{self.name}"
```

Comment:
Оскільки в рядку міститься лише одна поле класу то можна повертати безпосередньо його:
```suggestion
        return self.name
```